projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -870,7 +870,6 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy, CellT
             // while in edit mode subscribe to value changes on the current form control and set to editValue
             this.formControl.valueChanges.pipe(takeWhile(x => this.editMode)).subscribe(value => {
                 this.editValue = value;
```

Comment:
If this being a setter also goes and sets the value right back to the form control, wouldn't it sort of double dip with the validation logic? Wondering if we could use the form control as the backing for `editValue` entirely (assuming getting it from the service is quick), which might allow us to even skip this sync sub?